### PR TITLE
AutoCorrect fix files/zh-cn/web/**/* and fix spellcheck

### PR DESCRIPTION
**Pull request by huacnlee**
Ref: #4191 #3835

1. Add spaces between CJK and Alpha Letter or Numbers
2. Correct punctuations.
3. Fix some words spell.

- web -> Web
- Javascript, javascript -> JavaScript
- html, Html -> HTML
- css, Css -> CSS
- api -> API
- http, https -> HTTP, HTTPS

### CODE REVIEW

File: files/zh-cn/web/accessibility/an_overview_of_accessible_web_applications_and_widgets/index.html [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
+<p>web 正在变化。静态的、基于页面的站点逐渐被动态站点所取代，桌面式的 web 应用由大量的 JavaScript 和 AJAX 组成。设计人员完全可以通过 JavaScript，HTML 和 CSS 的组合创建令人惊叹的新的小部件和控件。这种转变有可能显着提高网络的响应能力和可用性，但是由于可访问性差距，存在许多用户有无法享用这种好处的风险。用户无法访问传统上 JavaScript 对于诸如屏幕阅读器等辅助技术，但是现在有了创建动态 Web 用户界面的方法，可以被各种用户访问。</p>
 
 <h2 id="问题">问题</h2>
 
-<p>大多数JavaScript工具包提供了模拟类似桌面界面行为的客户端小部件库。 滑块，菜单栏，文件列表视图等可以通过JavaScript，CSS和HTML的组合构建。 由于HTML 4规范不提供语义上描述这些窗口小部件的内置标签，因此开发人员通常会使用通用元素（如&lt;div&gt;和&lt;span&gt;）。 虽然这导致了一个看起来像桌面对应的小部件，但标记中通常没有足够的辅助技术可用的语义信息。 网页上的动态内容对于无论何种原因无法查看屏幕的用户来说都是特别有问题的。 股票行情，实时twitter消息更新，进度指示器和类似内容以辅助技术（AT）可能不知道的方式修改DOM。 那就是<a href="/en/ARIA" title="ARIA">ARIA</a>存在的意义。</p>
+<p>大多数 JavaScript 工具包提供了模拟类似桌面界面行为的客户端小部件库。 滑块，菜单栏，文件列表视图等可以通过 JavaScript，CSS 和 HTML 的组合构建。 由于 HTML 4 规范不提供语义上描述这些窗口小部件的内置标签，因此开发人员通常会使用通用元素（如&lt;div&gt;和&lt;span&gt;）。 虽然这导致了一个看起来像桌面对应的小部件，但标记中通常没有足够的辅助技术可用的语义信息。 网页上的动态内容对于无论何种原因无法查看屏幕的用户来说都是特别有问题的。 股票行情，实时 twitter 消息更新，进度指示器和类似内容以辅助技术（AT）可能不知道的方式修改 DOM。 那就是<a href="/en/ARIA" title="ARIA">ARIA</a>存在的意义。</p>
```
Comment (huacnlee) [inline files/zh-cn/web/accessibility/an_overview_of_accessible_web_applications_and_widgets/index.html:10]: twitter - Twitter
web - Web

**Comment by huacnlee** [comment]
This PR has to many files changed, GitHub Pull Request feature can't handle that, so I splited commits. 

Please review by click commit link to see the diffs.

File: files/zh-cn/web/svg/attribute/viewbox/index.html [hunk 2]
```diff
@@ -68,7 +68,7 @@
 <p>这个属性会受到 {{ SVGAttr("preserveAspectRatio") }} 的影响。</p>
 
 <div class="blockIndicator note">
-<p><strong>温馨提示:：</strong><code>width</code> 或者 <code>height</code> 的值，小于或等于0的情况下，这个元素将不会被渲染出来。</p>
```
Comment (huacnlee) [inline files/zh-cn/web/svg/attribute/viewbox/index.html:71]: Typo

File: files/zh-cn/web/javascript/a_re-introduction_to_javascript/index.html [hunk 3]
```diff
@@ -13,19 +13,19 @@
...
 
 <div class="note">
 <p><strong>备注：</strong> 由于这种用法更常见，从这里开始，我们将使用 JavaScript 来指代 ECMAScript 。</p>
 </div>
 
-<p>与大多数编程语言不同，JavaScript 没有输入或输出的概念。它是一个在宿主环境（host environment）下运行的脚本语言，任何与外界沟通的机制都是由宿主环境提供的。浏览器是最常见的宿主环境，但在非常多的其他程序中也包含 JavaScript 解释器，如 Adobe Acrobat、Adobe Photoshop、SVG 图像、Yahoo! 的 Widget 引擎，<a href="http://nodejs.org">Node.js</a> 之类的服务器端环境，NoSQL 数据库（如开源的 <a href="http://couchdb.apache.org">Apache CouchDB</a>）、嵌入式计算机，以及包括 <a href="http://www.gnome.org">GNOME</a> （注：GNU/Linux 上最流行的 GUI 之一）在内的桌面环境等等。</p>
+<p>与大多数编程语言不同，JavaScript 没有输入或输出的概念。它是一个在宿主环境（host environment）下运行的脚本语言，任何与外界沟通的机制都是由宿主环境提供的。浏览器是最常见的宿主环境，但在非常多的其他程序中也包含 JavaScript 解释器，如 Adobe Acrobat、Adobe Photoshop、SVG 图像、Yahoo！的 Widget 引擎，<a href="http://nodejs.org">Node.js</a> 之类的服务器端环境，NoSQL 数据库（如开源的 <a href="http://couchdb.apache.org">Apache CouchDB</a>）、嵌入式计算机，以及包括 <a href="http://www.gnome.org">GNOME</a> （注：GNU/Linux 上最流行的 GUI 之一）在内的桌面环境等等。</p>
```
Comment (huacnlee) [inline files/zh-cn/web/javascript/a_re-introduction_to_javascript/index.html]: `Yahoo！` has revert to `Yahoo!` in last commit.

File: files/zh-cn/web/javascript/about_javascript/index.html [hunk 4]
```diff
@@ -54,4 +54,4 @@ <h2 id="JavaScript_resources">JavaScript 资源</h2>
  <dd><a href="/zh-CN/docs/Web/JavaScript/Guide" title="zh-CN/docs/JavaScript/Guide">JavaScript 指南</a> 和 <a href="/zh-CN/docs/Web/JavaScript/Reference" title="zh-CN/docs/JavaScript/Reference">JavaScript 参考</a>。</dd>
 </dl>
 
-<p>JavaScript®是Oracle于美国和其他国家注册和拥有的商标。</p>
+<p>JavaScript®是 Oracle 于美国和其他国家注册和拥有的商标。</p>
```
Comment (huacnlee) [inline files/zh-cn/web/javascript/about_javascript/index.html:57]: This has fixed in last commit.

File: files/zh-cn/web/javascript/equality_comparisons_and_sameness/index.html [hunk 5]
```diff
@@ -32,7 +32,7 @@
 
 <p>选择使用哪个操作取决于你需要什么样的比较。</p>
 
-<p>简而言之，在比较两件事情时，双等号将执行类型转换; 三等号将进行相同的比较，而不进行类型转换 (如果类型不同, 只是总会返回 false );  而Object.is的行为方式与三等号相同，但是对于NaN和-0和+0进行特殊处理，所以最后两个不相同，而Object.is（NaN，NaN）将为 <code>true</code>。(通常使用双等号或三等号将NaN与NaN进行比较，结果为false，因为IEEE 754如是说.) 请注意，所有这些之间的区别都与其处理原语有关; 这三个运算符的原语中，没有一个会比较两个变量是否结构上概念类似。对于任意两个不同的非原始对象，即便他们有相同的结构， 以上三个运算符都会计算得到 false 。</p>
+<p>简而言之，在比较两件事情时，双等号将执行类型转换; 三等号将进行相同的比较，而不进行类型转换 (如果类型不同，只是总会返回 false );  而 Object.is 的行为方式与三等号相同，但是对于 NaN 和-0 和 +0 进行特殊处理，所以最后两个不相同，而 Object.is（NaN，NaN）将为 <code>true</code>。(通常使用双等号或三等号将 NaN 与 NaN 进行比较，结果为 false，因为 IEEE 754 如是说.) 请注意，所有这些之间的区别都与其处理原语有关; 这三个运算符的原语中，没有一个会比较两个变量是否结构上概念类似。对于任意两个不同的非原始对象，即便他们有相同的结构， 以上三个运算符都会计算得到 false 。</p>
```
Comment (huacnlee) [inline files/zh-cn/web/javascript/equality_comparisons_and_sameness/index.html:35]: `和-0` fixed in last commit.

**Comment by huacnlee** [comment]
@SphinxKnight @irvin

**Comment by irvin** [comment]
> This PR has to many files changed, GitHub Pull Request feature can't handle that, so I splitted commits.
> 
> Please review by click commit link to see the diffs.

this indeed contain too many files. can you make it into separate PR?

**Comment by SphinxKnight** [comment]
I've no right whatsoever to comment changes that only apply on `zh-TW` / `zh-CN` but it seems to me splitting this PR into multiple PRs (one for each section Web/HTML, Web/CSS, etc. Web/API A to <some letter>, etc.) would be more "digestible".

**Pull request closed without merging**